### PR TITLE
unit-test: Message.t.sol

### PR DESCRIPTION
**Pull request by odyslam**

### CODE REVIEW

File: packages/contracts-core/contracts/test/Message.t.sol [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    }
+
+    function test_body() public {
+        messageView = message.ref(0);
+        assertEq(Message.body(messageView).keccak(), body.ref(0).keccak());
+    }
+
+    function test_leaf() public {
+        messageView = message.ref(0);
+        assertEq(Message.leaf(messageView), messageHash);
```
Comment (prestwich) [inline packages/contracts-core/contracts/test/Message.t.sol:109]: method syntax is preferred

```
messageView.leaf()
```

**Comment by odyslam** [comment]
actually, let me do all

File: packages/contracts-core/contracts/test/Message.t.sol [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                recipient,
+                body
+            ),
+            messageHash
+        );
+    }
+
+    function test_origin() public {
+        messageView = message.ref(0);
+        assertEq(uint256(messageView.origin()), uint256(originDomain));
```
Comment (anna-carroll) [inline packages/contracts-core/contracts/test/Message.t.sol:79]: why casting them to `uint256`?  is this necessary for `assertEq` function?

**Review by anna-carroll — APPROVED** [review]
LGTM

File: packages/contracts-core/contracts/test/Message.t.sol (hunk 2 above)
Comment (odyslam) [reply packages/contracts-core/contracts/test/Message.t.sol:79]: yeah, at some point i want to PR some signatures to support other types of uints

**Review by prestwich — APPROVED** [review]

**Pull request merged**